src/main/java/com/powsybl/openloadflow/ac/TransformerVoltageControlOuterLoop.java

```diff
@@ -19,17 +19,20 @@
  */
 public class TransformerVoltageControlOuterLoop extends AbstractTransformerVoltageControlOuterLoop {
 
-    private double maxControlledNominalVoltage = Double.MIN_VALUE;
+    public static final String MAX_CONTROLLED_NOMINAL_VOLTAGE = "maxControlledNominalVoltage";
 
     @Override
     public void initialize(LfNetwork network) {
         // All transformer voltage control are disabled for the first equation system resolution.
+        double[] maxControlledNominalVoltage = new double[1];
```

Comment:
Why to you need an array here and not a simple double?

Reply:
To acces it from the lambda